control/state.py

```diff
@@ -172,9 +172,10 @@ def __init__(self, config):
         self.omap_name = f"nvmeof.{gateway_group}.state" if gateway_group else "nvmeof.state"
         ceph_pool = self.config.get("ceph", "pool")
         ceph_conf = self.config.get("ceph", "config_file")
+        rados_id = self.config.get_with_default("ceph", "id", "admin")
```

Comment:
nit: I am not sure about the default "admin". I would use an empty string (so we would use the rados lib's default). Or at least have `id = admin` in the example ceph-nvmeof.conf to match the default.

Though this is just IMHO and I am fine with the current version too. Let's see may be other people have an opinion about this.

Reply:
that makes sense to me. I've changed the default to be an empty string and we'll just rely on rados lib's default.